src_fun/main.cpp

```diff
@@ -3,46 +3,110 @@
 #include "../src_common/hash.h"
 #include "../src_common/integer.h"
 #include "api.h"
+#include "testing.h"
+
 
 EnterApplicationNamespace
 
 
+static const String TEST_DIRECTORY = "test"_s;

```

Comment:
After doing the merges below, this will be used in one place and there is no need for this loud constant then.